template/templates/views/feed_entries.html

```diff
@@ -17,6 +17,9 @@ <h1 dir="auto">
                 data-label-loading="{{ t "confirm.loading" }}"
                 data-show-only-unread="{{ if .showOnlyUnreadEntries }}1{{ end }}">{{ t "menu.mark_page_as_read" }}</a>
         </li>
+        <li>
+            <a href="{{ route "markFeedAsRead" "feedID" .feed.ID }}?redirect_to_self=true">{{ t "menu.mark_all_as_read" }}</a>
```

Comment:
I just realized that there is no confirmation yes/no. The unread page has one.

Reply:
Indeed. This led me down a small rabbithole reading the code about how that is implemented figuring out that there's more to mark-all-as-read 's behavior. Notably #1064. I 've sent PR #1065 to fix that. I 'll then rebase this PR on top of that and switch to the same implementation here (minus the tickbox)